SuperBuild/External_vtkRenderingVR.cmake

```diff
@@ -90,6 +93,8 @@ if(NOT DEFINED ${proj}_DIR AND NOT ${SUPERBUILD_TOPLEVEL_PROJECT}_USE_SYSTEM_${p
       # Required to find VTK
       -DVTK_DIR:PATH=${VTK_DIR}
       ${EXTERNAL_PROJECT_OPTIONAL_CMAKE_CACHE_ARGS}
+      # Required to find vtkRenderingVRModels
+      -DvtkRenderingVRModels_DIR:PATH=${vtkRenderingVRModels_DIR}
```

Comment:
I just realized that we should add the version condition here as well. However, I'm not sure how you typically do it in such case @jcfr